src/state/queries/resolve-identity.ts

```diff
@@ -0,0 +1,27 @@
+import {LRU} from './direct-fetch-record'
+
+// TODO: lift this into direct fetch to share cache
```

Comment:
can we drop this comment since this PR achieves this?

Reply:
Thought about doing that, but wasn't sure if that was what that comment meant, will do!